SITnett/templates/produksjoner/produksjon_info.html

```diff
@@ -74,18 +87,92 @@ <h2>{{produksjon.tittel}}</h2>
 		{% if vervoppslag %}
 			{% for verv, erfaringer in vervoppslag.items %}
 				{% if FEATURES.TOGGLE_VERV and verv.erfaringsoverforing and user.is_authenticated %}
-					{% if erfaringer.count > 1 %}
-						<a href="{% url 'verv_info' verv.id %}"><b>{{verv.plural|capfirst}}:</b></a><br>
+					<a href="{% url 'verv_info' verv.id %}"><b>
+				{% else %}
+					<b>
+				{% endif %}
+				{% if erfaringer.count > 1 %}
+					{% if produksjon.UKEtype == "UKErevy" %}
+						{% if verv.tittel == "forfatter" %}
+							Forfatterkollegiet:
+						{% elif verv.tittel == "komponist" %}
+							Komponistkollegiet:
+						{% elif verv.tittel == "musiker" %}
+							Revybandet:
+						{% elif verv.tittel == "lysdesigner" %}
+							Lysmestere:
+						{% elif verv.tittel == "lystekniker" %}
+							Lyslaget:
+						{% elif verv.tittel == "videodesigner" %}
+							Videomestere:
+						{% elif verv.tittel == "videotekniker" %}
+							Videolaget:
+						{% elif verv.tittel == "lysreklamist" %}
+							Lysreklamen:
+						{% else %}
+							{{verv.plural|capfirst}}:
+						{% endif %}
+					{% elif produksjon.UKEtype == "UKEintim" %}
+						{% if verv.tittel == "forfatter" %}
+							Forfatterkollegiet:
+						{% elif verv.tittel == "komponist" %}
+							Komponistkollegiet:
+						{% elif verv.tittel == "musiker" %}
+							Bandet:
+						{% else %}
+							{{verv.plural|capfirst}}:
+						{% endif %}
+					{% elif produksjon.UKEtype == "supperevy" %}
+						{% if verv.tittel == "produsent" %}
+							Suppedirektører:
+						{% elif verv.tittel == "regissør" %}
+							Supperegissører:
+						{% elif verv.tittel == "lystekniker" %}
+							Suppelys:
+						{% elif verv.tittel == "lydtekniker" %}
+							Suppelyd:
+						{% else %}
+							{{verv.plural|capfirst}}:
+						{% endif %}
```

Comment:
Dette funker fint her, men kan bli litt pes om vi har lyst til å gjøre lignende justeringer i andre sider senere, og det er også litt pes at akkurat det same skjer lenger ned i filen. Kan vurdere om dette passer bedre å implementere i `models.py`. Kanskje det er en idé å bruke en [custom model manager](https://docs.djangoproject.com/en/3.1/topics/db/managers/)?

Reply:
Ja, det er mulig det er bedre å putte det i models.py. Da må i så fall Erfaring gjøre oversettelsene, ettersom de er basert på både Verv (hvilket verv det er snakk om) og Produksjon (om produksjonen er en UKEproduksjon). Jeg skal se på det nå.

Det som skjer lenger ned er entallsbøying av de samme vervtitlene. Siden slik bøying er såpass uregelmessig tror jeg det blir vanskelig å unngå to tabeller av denne typen.

Reply:
Vi bør endre dette slik at oppslaget skjer i modellen, ja, men det er litt mer dyptgående, så jeg foreslår å merge inn dette midlertidig, og så skal jeg ordne en bedre løsning til neste helg. Da kan jeg fikse bedre rekkefølge og visning av erfaringer i samme slengen.